frontend/components/Layouts/DashboardLayout.tsx

```diff
@@ -16,7 +16,24 @@ interface Props {
 const DashboardLayout: React.FC<Props> = ({ currentUser, children, withPadding = true }) => {
   const { width } = useWindowSize()
   const router = useRouter()
-  const [navExpanded, setNavExpanded] = useState(false)
+  const shouldNavBeExpanded = () => {
```

Comment:
This logic used to be only in the `useEffect`, but I extracted it out into its own function so that we can also use it for the initial value of the `navExpanded` `useState`